src/database/models/user.ts

```diff
@@ -56,9 +56,9 @@ export default class User extends Model {
       as: 'role'
     });
 
-    User.belongsTo(University, {
-      foreignKey: 'universityId',
-      targetKey: 'id',
+    User.hasOne(University, {
```

Comment:
I don't think this is doing what you expect it to do
I think this is adding an extra foreign key on the university

A user does belong to a university so I don't know why this would change